proto/pairing/provider_payment_storage.proto

```diff
@@ -6,11 +6,10 @@ import "pairing/unique_payment_storage_client_provider.proto";
 
 message ProviderPaymentStorage {
   string index = 1; 
-  repeated UniquePaymentStorageClientProvider uniquePaymentStorageClientProvider = 2; 
-  uint64 epoch = 3; 
+  repeated string uniquePaymentStorageClientProviderKeys = 2; 
```

Comment:
dont use the same protobuf number, reserve 2 and make it the next available (5)

Reply:
done